app/src/main/res/values-es-rES/strings.xml

```diff
@@ -138,7 +138,7 @@
 
     <!-- This string is used to label a button in the 'Settings' dialog window that, when pressed,
         opens a dialog box that contains display-related settings: window size, pixel density, etc. -->
-    <string name="settings_display">Mostrar</string>
+    <string name="settings_display">Opciones de visualización</string>
```

Comment:
aren't these strings auto-generated from Pontoon/the latest L10n system?

@Pike: if so, do we need to update this string in the central source of truth as well?